guides/micronaut-mqtt/cli/java/src/main/java/example/micronaut/MicronautguideCommand.java

```diff
@@ -25,29 +27,31 @@
 
     @Option(names = {"-s", "--scale"}, // <2>
             required = false, // <3>
-            description = "Temperate scales ${COMPLETION-CANDIDATES}",  // <4>
-            completionCandidates = TemperatureScaleCandidates.class)  // <4>
+            description = "Temperate scales ${COMPLETION-CANDIDATES}", // <4>
+            completionCandidates = TemperatureScaleCandidates.class) // <4>
     String scale;
 
     @Inject
-    TemperatureClient temperatureClient;  // <5>
+    TemperatureClient temperatureClient; // <5>
 
-    public static void main(String[] args) throws Exception {
+    public static void main(String[] args) {
         PicocliRunner.run(MicronautguideCommand.class, args);
     }
 
     public void run() {
         Scale temperatureScale = scale != null ?
-                Scale.of(scale).orElse(Scale.CELSIUS) : Scale.CELSIUS;
-        BigDecimal celsius = (temperatureScale == Scale.FAHRENHEIT)
+                Scale.of(scale).orElse(CELSIUS) : CELSIUS;
+        BigDecimal celsius = (temperatureScale == FAHRENHEIT)
                 ? fahrenheitToCelsius(temperature) : temperature;
         byte[] data = celsius.toPlainString().getBytes();
         temperatureClient.publishLivingroomTemperature(data); // <6>
-        System.out.println("Topic published");
```

Comment:
Do we recommend for CLI applications to use LOG statements for command output?. The generated CLI app in Launch uses `System.out.println` for command output.